00_download_data/nbconverted/2_sample_intersection.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Explore sample intersection between TCGA data types
+# 
+# As we add more data types to our classification experiments, we've been filtering to only the set of samples that have data for each included data type. We want to know more about how many samples and which/how many cancer types are affected by this filtering, since it could influence the results in subtle ways.
+
+# In[1]:
+
+
+import os
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+import mpmp.config as cfg
+
+
+# In[2]:
+
+
+# get sample list for each data type
+sample_lists = {}
+for training_data, sample_info_file in cfg.sample_infos.items():
+    samples = pd.read_csv(sample_info_file, sep='\t', index_col=0).index
+    sample_lists[training_data] = set(samples)
+
+
+# In[3]:
+
+
+# counts per data type
+print('\n'.join(['{}\t{}'.format(n, len(v)) for n, v in sample_lists.items()]))
+
+
+# We'll start by just counting the number of samples that have data for each of the relevant data types.
+
+# In[4]:
+
+
+# do venn diagram
+from venn import venn
```

Comment:
You may want to move this import to the top since you end up importing it multiple times